### PR TITLE
Allow dmrelay to only be used in mod channels.

### PR DESCRIPTION
**Pull request by Qwerty-133**
DMs between bot and users is private information which mods shouldn't be able to relay in public channels.

### CODE REVIEW

File: bot/exts/moderation/dm_relay.py [hunk 1]
```diff
@@ -64,7 +65,8 @@ async def dmrelay(self, ctx: Context, user: discord.User, limit: int = 100) -> N
 
     async def cog_check(self, ctx: Context) -> bool:
         """Only allow moderators to invoke the commands in this cog."""
```
Comment (mbaruh) [inline bot/exts/moderation/dm_relay.py:67]: Should update the comment here.

Comment (Qwerty-133) [reply bot/exts/moderation/dm_relay.py]: Done, thanks!

**Review by Xithrius — APPROVED** [review]
Does the do.

**Review by ChrisLovering — APPROVED** [review]
Looks good 👍

**Pull request merged**